database/migrations/2022_03_18_104400_add_total_to_consumables_users.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
This one either

Reply:
this one is the migration script for my feature, so I keep it here